content/en/docs/collector/building/authenticator-extension.md

```diff
@@ -29,13 +29,13 @@ room at the [CNCF Slack workspace](https://slack.cncf.io).
 
```

Comment:
I can't leave a suggestion for this, but can you edit the above paragraphs to say:

```
[API Reference Guide](https://pkg.go.dev/go.opentelemetry.io/collector/config/configauth)
for the semantics of each type and function.
If you need help, join the
[#opentelemetry-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W)
channel at the [CNCF Slack workspace](https://slack.cncf.io).
```